mastering/data/instance_names.csv

```diff
@@ -1,65 +1,65 @@
-Recursive Mono,Casual Italic,28,13,RecursiveMono-CasualItalic,26,Recursive Mn Cs,Italic,15,6,22
-Recursive Sans,Casual Italic,28,13,RecursiveSans-CasualItalic,26,Recursive Sn Cs,Italic,15,6,22
-Recursive Mono,Casual Light Italic,34,19,RecursiveMono-CasualLtItal,26,Recursive Mn Cs Lt,Italic,18,6,25
-Recursive Mono,Medium Italic,28,13,RecursiveMono-MediumItalic,26,Recursive Mono Med,Italic,18,6,25
-Recursive Sans,Casual Light Italic,34,19,RecursiveSans-CasualLtItal,26,Recursive Sn Cs Lt,Italic,18,6,25
-Recursive Sans,Medium Italic,28,13,RecursiveSans-MediumItalic,26,Recursive Sans Med,Italic,18,6,25
-Recursive Mono,Casual SemiBold Italic,37,22,RecursiveMono-CasualSmBdIt,26,Recursive Mn CsSmBd,Italic,19,6,26
-Recursive Mono,SemiBold Italic,30,15,RecursiveMono-SemiBdItalic,26,Recursive Mono SmBd,Italic,19,6,26
-Recursive Sans,Casual SemiBold Italic,37,22,RecursiveSans-CasualSmBdIt,26,Recursive Sn CsSmBd,Italic,19,6,26
-Recursive Sans,SemiBold Italic,30,15,RecursiveSans-SemiBdItalic,26,Recursive Sans SmBd,Italic,19,6,26
-Recursive Mono,Casual SemiBold,30,15,RecursiveMono-CasualSemiBd,26,Recursive Mn CsSmBd,Regular,19,7,27
-Recursive Sans,Casual SemiBold,30,15,RecursiveSans-CasualSemiBd,26,Recursive Sn CsSmBd,Regular,19,7,27
-Recursive Mono,Casual Regular,29,14,RecursiveMono-CasualRegular,27,Recursive Mn Cs,Regular,15,7,23
-Recursive Sans,Casual Regular,29,14,RecursiveSans-CasualRegular,27,Recursive Sn Cs,Regular,15,7,23
-Recursive Mono,ExtraBold Italic,31,16,RecursiveMono-ExtraBdItalic,27,Recursive Mono XBd,Italic,18,6,25
-Recursive Mono,UltraBold Italic,31,16,RecursiveMono-UltraBdItalic,27,Recursive Mn UltBd,Italic,18,6,25
-Recursive Sans,ExtraBold Italic,31,16,RecursiveSans-ExtraBdItalic,27,Recursive Sans XBd,Italic,18,6,25
-Recursive Sans,UltraBold Italic,31,16,RecursiveSans-UltraBdItalic,27,Recursive Sn UltBd,Italic,18,6,25
-Recursive Mono,Casual Black Italic,34,19,RecursiveMono-CasualBlkItal,27,Recursive Mn Cs Blk,Italic,19,6,26
-Recursive Mono,Casual UltraBold Italic,38,23,RecursiveMono-CasualUltBdIt,27,Recursive MnCsUltBd,Italic,19,6,26
-Recursive Sans,Casual Black Italic,34,19,RecursiveSans-CasualBlkItal,27,Recursive Sn Cs Blk,Italic,19,6,26
-Recursive Sans,Casual UltraBold Italic,38,23,RecursiveSans-CasualUltBdIt,27,Recursive SnCsUltBd,Italic,19,6,26
-Recursive Mono,Casual ExtraBold,31,16,RecursiveMono-CasualExtraBd,27,Recursive Mn Cs XBd,Regular,19,7,27
-Recursive Mono,Casual UltraBold,31,16,RecursiveMono-CasualUltraBd,27,Recursive MnCsUltBd,Regular,19,7,27
-Recursive Sans,Casual ExtraBold,31,16,RecursiveSans-CasualExtraBd,27,Recursive Sn Cs XBd,Regular,19,7,27
-Recursive Sans,Casual UltraBold,31,16,RecursiveSans-CasualUltraBd,27,Recursive SnCsUltBd,Regular,19,7,27
-Recursive Mono,Casual Bold Italic,33,18,RecursiveMono-CasualBdItal,26,Recursive Mn Cs,Bold Italic,15,11,27
\ No newline at end of file
+Recursive,Mono Casual SemiBold Italic,37,27,RecursiveMono-CslSmBdItalic,27,Recursive Mn Csl SmBd,Italic,21,6,28

```

Comment:
I thought we were going to try to split the statics into at least two families: Recursive Mono & Recursive Sans. Does this change indicate that this will just be in a single family?

Reply:
This does it. The first two entries are looking at how the instances are named in the designspace file as a key. 

This, however, isn't breaking it into 4 families, as I think we decided to do. Will fix

Reply:
Sorry, to be more clear: the script needs to find the instances in the designspace. Your named instances are in the format of "Recursive" for the family name, and then the style name. I don't think you can break named instances in to sub-families, afaik. I'll add clarity to the CSV

Reply:
Ah, got it! Thanks for explaining that.

Yes, 4 families will probably work the best, so font menus are as simple as possible for people to navigate, and so it corresponds to what will (probably) be available on Google Fonts.